### PR TITLE
4166 data loader bc comp

### PR DESCRIPTION
**Pull request by kialj876**
*Issue #:* /bcgov/entity#4166

*Description of changes:*
- updated data loader for loading in bc companies
- refactored to be more usable, added linting etc.
- updated colin-api for some issues when getting bc companies
- updated insert/get on share structure class values (they were incorrect before - waiting for clarification from David and or Linda before merge)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: lear-db/test_data/requirements/bcregistry-libraries.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+git+https://github.com/bcgov/business-schemas.git@2.5.11#egg=registry_schemas
```
Comment (thorwolpert) [inline lear-db/test_data/requirements/bcregistry-libraries.txt:1]: Should this move to the current version??

Comment (kialj876) [reply lear-db/test_data/requirements/bcregistry-libraries.txt:1]: Oh nice catch - I will update this to always get the current one

File: colin-api/src/colin_api/models/corp_party.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 from flask import current_app
 
 from colin_api.exceptions import PartiesNotFoundException
-from colin_api.models import Address  # pylint: disable=cyclic-import
+from colin_api.models import Address, Business
 from colin_api.resources.db import DB
```
Comment (thorwolpert) [inline colin-api/src/colin_api/models/corp_party.py:26]: move the DB before the models, as they import it.

Comment (kialj876) [reply colin-api/src/colin_api/models/corp_party.py:26]: thanks, was wondering what to do here

Comment (kialj876) [reply colin-api/src/colin_api/models/corp_party.py:25]: hmm it still doesn't like it so I'm going to put it back and add the pylint disable for now (it was there before - I don't remember why, but I took it off in my changes)

Comment (kialj876) [reply colin-api/src/colin_api/models/corp_party.py:27]: alright changed it again haha - this was a tricky one because it wasn't coming up locally

**Pull request merged**